lib/modules/platform/github/index.ts

```diff
@@ -1694,13 +1694,21 @@ async function tryPrAutomerge(
         { prNumber, errors: res.errors },
         'GitHub-native automerge: fail',
       );
+      if (platformPrOptions.automergeFailureComment === 'on-error') {
```

Comment:
Wrong file for implementation. This only works for GitHub, and even then only for GitHub- active automerge.